docker/README.md

```diff
@@ -0,0 +1,54 @@
+--------
+# Docker for BigDL
+
+## How to build it.
+
+### By default, you can build a bigdl:default image with latest nightly-build BigDL distributions:
+
+    sudo docker build --rm -t intelanalytics/bigdl:default .
```

Comment:
just use "bigdl:default"?

Reply:
we need a repository name, if it is without "intelanalytics", it will be a official image maintained by docker company.

Reply:
or we can use bigdl as repository name like
bigdl/bigdl:default
bigdl/bigdl:0.2.0-spark-2.1.0
this requires us to register a bigdl account in the Docker hub.
I have already have intelanalytics account registered with my intel.com email. please ask some one who still have free email to register the bigdl account.
Thanks.

Reply:
we can use bigdl